pandas/core/indexes/base.py

```diff
@@ -4902,7 +4902,9 @@ def isin(self, values, level=None):
 
         Compute boolean array of whether each index value is found in the
         passed set of values. The length of the returned boolean array matches
-        the length of the index.
+        the length of the index. If the index is a multi-index and no level is
+        given, the values must be given as list containing tuples with the same
+        length as the mulit-index.
```

Comment:
```suggestion
        given, the values must be given as list containing tuples of the same
        length as the number of levels in the MultiIndex
```

Do the tuples not have to be the same length as the number of levels in the MultiIndex, rather than to its length?

Reply:
Hey, thanks. That was what I meant. Your wording is way better.